internal/ethapi/simulate.go

```diff
@@ -73,6 +73,14 @@ func (r *simCallResult) MarshalJSON() ([]byte, error) {
 	return json.Marshal((*callResultAlias)(r))
 }
 
+// simBlockResult is the result of a simulated block.
+type simBlockResult struct {
+	fullTx      bool
+	chainConfig *params.ChainConfig
+	Block       *types.Block
+	Calls       []simCallResult
+}
+
```

Comment:
When resolving conflicts I chose to omit the `MarshalJSON()` method from upstream here. I reintroduced it later.

Reply:
I plan on rewriting the history of this branch to remove this meander.